minifi-bootstrap/src/main/java/org/apache/nifi/minifi/bootstrap/util/ConfigTransformer.java

```diff
@@ -219,6 +223,7 @@ private static void writeNiFiProperties(Map<String, Object> topLevelYaml, Output
             Map<String, Object> componentStatusRepo = (Map<String, Object>) topLevelYaml.get(COMPONENT_STATUS_REPO_KEY);
             Map<String, Object> securityProperties = (Map<String, Object>) topLevelYaml.get(SECURITY_PROPS_KEY);
             Map<String, Object> sensitiveProperties = (Map<String, Object>) securityProperties.get(SENSITIVE_PROPS_KEY);
+            Map<String, Object> provenanceProperties = (Map<String, Object>) securityProperties.get(PROVENANCE_REPORTING_KEY);
```

Comment:
Upon further inspection I see this is unused and the provenance properties are gathered in createFlowXml.  Likely just remove unless you would prefer to gather these maps up front.